services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -89,23 +90,38 @@ export const SubmissionSummary = (): React.ReactElement => {
     // This is a hacky way to fake feature flags before we have feature flags.
     // please avoid reading env vars outside of index.tsx in general. 
     const environmentName = process.env.REACT_APP_STAGE_NAME || ''
-    const isProdEnvironment = ['prod', 'val', 'nope'].includes(environmentName)
+    const isProdEnvironment = ['prod', 'val'].includes(environmentName)
 
     const displayUnlockButton = !isProdEnvironment && loggedInUser?.role === 'CMS_USER'
 
     useEffect(() => {
-        updateHeading(pathname, 'TEMP NAME')
-    }, [updateHeading, pathname])
+        if (submissionAndRevisions) {
```

Comment:
Looks good this is better. Still would like to see us abstract the api request + data transformation  farther from the React UI  code  as we migrate the  other apis.

Reply:
I’d love to see your thoughts. Should we wrap up the use…Query() call in something that handles that for us? In general I’m a fan of having a package that wraps up a dependency.